releases/latest/postgresql-k8s-bundle.yaml

```diff
@@ -1,20 +1,21 @@
+bundle: kubernetes
+name: postgresql-k8s-bundle
+type: bundle
 applications:
   pgbouncer-k8s:
-    channel: edge
     charm: pgbouncer-k8s
+    channel: edge
+    scale: 1
     constraints: arch=amd64
     revision: 7
-    scale: 1
   postgresql-k8s:
-    channel: edge
     charm: postgresql-k8s
+    channel: edge
     constraints: arch=amd64
     revision: 24
     scale: 1
     trust: true
-bundle: kubernetes
-name: postgresql-k8s-bundle
+    revision: 22
```

Comment:
did not check this, but wouldn't not setting a revision get the latest one?

Reply:
checked [here](https://juju.is/docs/sdk/set-charm-channels-within-a-bundle) - Isn't this the way to go?

Reply:
Revisions don't actually do anything - the idea here is that every charm update should trigger a PR, which runs tests, meaning we have a log in the CI of every charm update.